package.json

```diff
@@ -10,7 +10,7 @@
   "type": "module",
   "scripts": {
     "lint": "eslint .",
-    "format": "prettier . --write",
+    "format": "eslint --fix .",
```

Comment:
Just out of curiosity, why the switch?

Reply:
The whole idea behind integrating Prettier into ESLint is for it to inherit its ruleset while allowing us to configure how these rules are applied. It's generally much better to call a single linter as opposed to maintaining potentially diverging configurations across multiple linters. For instance, if your `.prettierignore` was different from the files ignored by ESLint, then calling `prettier --write` and `eslint --fix` may produce different results.